index.yaml

```diff
@@ -5314,6 +6099,24 @@ entries:
     - assets/rancher-sriov/sriov-crd-100.0.0+up0.1.0.tgz
     version: 100.0.0+up0.1.0
   system-upgrade-controller:
+  - annotations:
+      catalog.cattle.io/certified: rancher
+      catalog.cattle.io/hidden: "true"
+      catalog.cattle.io/namespace: cattle-system
+      catalog.cattle.io/os: linux
+      catalog.cattle.io/release-name: system-upgrade-controller
```

Comment:
The annotations for `system-upgrade-controller-100.0.1` are out of sync with the chart.

Reply:
```
Checking annotations on chart charts/system-upgrade-controller/system-upgrade-controller/100.0.1+up0.3.1
ERROR: Contents of Chart.yaml for key 'appVersion' differs from index.yaml

Expected:
v0.8.1
Found:
v0.8.0

ERROR: Contents of Chart.yaml for key 'annotations' differs from index.yaml

Expected:
catalog.cattle.io/certified: rancher
catalog.cattle.io/hidden: "true"
catalog.cattle.io/kube-version: '>=1.18.0-0'
catalog.cattle.io/namespace: cattle-system
catalog.cattle.io/os: linux
catalog.cattle.io/rancher-version: '>= 2.6.3-alpha'
catalog.cattle.io/release-name: system-upgrade-controller
Found:
catalog.cattle.io/certified: rancher
catalog.cattle.io/hidden: "true"
catalog.cattle.io/namespace: cattle-system
catalog.cattle.io/os: linux
catalog.cattle.io/release-name: system-upgrade-controller
```
confirmed by @aiyengar2 output script

Reply:
addressed by https://github.com/rancher/charts/pull/1671